spec/spec_helper.rb

```diff
@@ -13,6 +13,7 @@
 
 $LOAD_PATH.unshift "lib"
 require "fog/core"
+require "fog/mock"
```

Comment:
This sounds amazing in the final user tests. =)

Just a plain `require 'fog-mock'` at the beginning of the test helper. 👍